### PR TITLE
chore(#50): change the build action target to integrationTest 

### PR DESCRIPTION
**Pull request by eharrow**
from test as unit tests do not provide sufficient coverage.  Note that this will fail quality checks as the build has a failed test.

### CODE REVIEW

**Comment by eharrow** [comment]
Build fails as expected.  Either @phughk merge anyway or we fix the root cause - the failed test.  At least now we can have some confidence.

**Comment by phughk** [comment]
This is fantastic, thank you!
Ideally I would like `gradle test` to run unit and `gradle integration` to run integration tests. The github workflow needs to change I think, but also the gradle tasks. Either of us could do that, but I am quite occupied with live queries atm. 
Will think about landing this if it isn't addressed, but if it can be addressed then that would be 10/10 amazing.
Thanks for this - we definitely need this change :)

**Comment by phughk** [comment]
Great thank you! Will land this once #53 is resolved 🎉

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Run Tests
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: test
+          arguments: integrationTest
```
Comment (phughk) [inline .github/workflows/test.yml:30]:
```suggestion
          arguments: test integrationTest
```

**Comment by phughk** [comment]
Thanks for this @eharrow ! 🎉

**Pull request merged**